app/controllers/manage_assessments/dashboard_controller.rb

```diff
@@ -2,5 +2,9 @@ class ManageAssessments::DashboardController < ApplicationController
   def show
     @courses = policy_scope(Course.includes(:outcomes_with_metadata))
     authorize(:generic, :view_assessments?)
+
+    if @courses.count == 1
```

Comment:
We should probably use `.length` here instead of `.count`. The difference is subtle and small in this case, but `.count` will issue a query returning only the count of records, while `.length` will get all of the records and then check the length of the array.

We need all of the records whether the count is 0, 1, or 10, so we might as well just load it up to avoid doing the `count` query and then the full query immediately after.